### PR TITLE
--lazy is only available with -l in trusty.

### PR DESCRIPTION
**Pull request by vosst**

### CODE REVIEW

**Comment by zyga** [comment]
+1

File: overlord/snapstate/backend/mountunit.go [hunk 1]
```diff
@@ -61,7 +61,7 @@ func removeMountUnit(baseDir string, meter progress.Meter) error {
 	if osutil.FileExists(unit) {
 		// use umount --lazy to ensure that even busy mount points
 		// can be unmounted
```
Comment (chipaca) [inline overlord/snapstate/backend/mountunit.go]: maybe clarify `--lazy` → `-l`

File: overlord/snapstate/backend/mountunit.go [hunk 2]
```diff
@@ -59,8 +59,9 @@ func removeMountUnit(baseDir string, meter progress.Meter) error {
 	sysd := systemd.New(dirs.GlobalRootDir, meter)
 	unit := systemd.MountUnitPath(dirs.StripRootDir(baseDir), "mount")
 	if osutil.FileExists(unit) {
-		// use umount --lazy to ensure that even busy mount points
-		// can be unmounted
+		// use umount -l (lazy) to ensure that even busy mount points
+		// can be unmounted.
+		// note that the long option --lazy is not supported on trusty.
```
Comment (chipaca) [inline overlord/snapstate/backend/mountunit.go]: nice

**Comment by mvo5** [comment]
:+1: lets merge once tests are green.

**Pull request merged**